.github/workflows/ci.yml

```diff
@@ -27,6 +27,9 @@ jobs:
         with:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
+      - run: julia --project -e 'using Pkg; Pkg.develop([PackageSpec(path="SnoopCompileCore"),
```

Comment:
I wonder if this needs some escaping on Windows?

```

Run julia --project -e 'using Pkg; Pkg.develop([PackageSpec(path="SnoopCompileCore"), PackageSpec(path="SnoopCompileAnalysis"), PackageSpec(path="SnoopCompileBot")])'
  julia --project -e 'using Pkg; Pkg.develop([PackageSpec(path="SnoopCompileCore"), PackageSpec(path="SnoopCompileAnalysis"), PackageSpec(path="SnoopCompileBot")])'
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
ERROR: UndefVarError: SnoopCompileCore not defined
Stacktrace:
 [1] top-level scope at none:0
```

Reply:
```
julia --project -e 'using Pkg; Pkg.develop([PackageSpec(path=\"SnoopCompileCore\"),
                                            PackageSpec(path=\"SnoopCompileAnalysis\"),
                                            PackageSpec(path=\"SnoopCompileBot\")])'
```
works fine for me on Windows.

Reply:
This is becayse PowerShell needs `\` for scaping `"`. Instead, we can add the following before `jobs`. It will use bash for all operating systems:
```
defaults:
  run:
    shell: bash
```

Reply:
I'm not even sure how to interpret that error message. Anyone have any ideas?

Reply:
@timholy Let's first choose bash as I suggested. Then we can see if it errors again.

Reply:
Oh, looks like I just needed to refresh my browser.